setup.py

```diff
@@ -43,6 +43,7 @@ def get_docs_url():
         "pysoem==1.0.7",
         "numpy==1.19.5",
         "scipy==1.5.4",
+        "dataclasses==0.6",
```

Comment:
Does this requirement work outside of python 3.6 or ignored somehow? I think it's just a backport of 3.7 std.

Reply:
I'll remove this requirement. I'm not even using dataclasses :)